deps/CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ option(USE_BUNDLED_DEFAULT
         "Set default option value for bundled dependencies"
         ${DEFAULT_ENABLED_BUNDLED_DEPS})
 
+option(USE_BUNDLED_LIBWHEREAMI
+    "Use bundled libwhereami"
+    ON)
+
 # Shorthand for declaring an option and adding a subdirectory if option is on
```

Comment:
Actually, I was wondering that maybe now it would be a better idea to just make all bundles default to ON on all OSes, what do you think?